src/routes.js

```diff
@@ -128,7 +138,7 @@ export default [
     }
   },
   {
-    path: `/create/password`,
+    path: `/create/:networkId/password`,
```

Comment:
why only add `networkId` to password and create?

#3213 ?

Reply:
Didn't you have something on this? Mind to push a PR?

Reply:
These routes are now back to "normal". I went for a completely different solution ([see](https://github.com/luniehq/lunie/pull/3439#discussion_r368981255))